src/app/modules/main/chat_section/color_hash_model.nim

```diff
@@ -0,0 +1,53 @@
+import NimQml, Tables
+
+import color_hash_item
+
+type
+  ModelRole {.pure.} = enum
+    SegmentLength = UserRole + 1
+    ColorId
+
+QtObject:
+  type
+    Model* = ref object of QAbstractListModel
+      items*: seq[Item]
```

Comment:
@osmaczko can we have `items` as private?

Reply:
Yeah sure, no point to make it public, good catch, thanks.